### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by differentreality**
Some updates to our documentation, added more info about commits in contributing guide, and included labels.

I have to add a description for a couple of more labels - I don't recall how we came up with labels ready and in progress.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# Request for contributions
+# Contributing to OSEM
+Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) and send a PR!
```
Comment (Ana06) [inline CONTRIBUTING.md]: It is a little bit confusing as it seems that you need to create a issue to open a PR. I would also prefer not ot abbreviate PR. what about this:

Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) or work in one if the [open issues](https://github.com/openSUSE/osem/issues). Then open a Pull Request!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,14 +62,14 @@ You can access the app [localhost:3000](http://localhost:3000). Whatever you cha
     vagrant exec bundle exec rake db:migrate
     ```
 
-## How to contribute
+## How to contribute code
```
Comment (Ana06) [inline CONTRIBUTING.md]: How to contribute **with** code

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -79,17 +84,33 @@ You can access the app [localhost:3000](http://localhost:3000). Whatever you cha
...
+
 ### Pull Requests workflow
-Please open a PR only when you have finished coding, and your changes are ready to be reviewed for merging.
+Please open a pull request (PR) only when you have finished coding, and your changes are ready to be reviewed for merging.
 
 * Title
   * Include a comprehensive title about what this PR is doing
   * Referencing the issue number on the PR title is not giving any information about what this PR is about
-  * The title should be short (50 characters maximum). You can add more information in the description
+  * The title should be short (50 characters maximum); you can add more information in the description
```
Comment (Ana06) [inline CONTRIBUTING.md]: To be honest I was thinking about the commit and not the PR when writing this, as you already added in the commit section it can be removed.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -120,11 +141,15 @@ We are using [rspec](http://rspec.info/)+[capybara](http://jnicklas.github.io/ca
 vagrant exec bundle exec rspec
 ```
 
-## Code of Conduct
```
Comment (Ana06) [inline CONTRIBUTING.md]: Why did you remove this section?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -120,11 +141,15 @@ We are using [rspec](http://rspec.info/)+[capybara](http://jnicklas.github.io/ca
...
+### Review App of your PR
+
+OSEM uses [Review Apps](https://devcenter.heroku.com/articles/github-integration-review-apps) on Heroku.
+
+* The review app can be manually created by a maintainer, and when that happens you will see a relevant message in the PR
+
+* Please help reviewers by adding the necessary data relevant to your PR,
+eg. if your PR is doing something related to conference registrations, go to the review app and make sure there is a conference with registrations.
 
-## Contact
```
Comment (Ana06) [inline CONTRIBUTING.md]: Why did you remove this section?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -144,3 +169,44 @@ config.ichain_force_test_username = "testuser"
 # set email of 'testuser'
 config.ichain_force_test_attributes = {:email => "testuser@example.com"}
 ```
+## Labels for issues and PRs
```
Comment (Ana06) [inline CONTRIBUTING.md]: Great! This is really useful! :tada: 

Although, what are there some tags without description?

File: README.md [hunk 7]
```diff
@@ -10,6 +10,8 @@
 
 An event management tool tailored to Free and Open Source Software conferences.
 
+OSEM actively participates in [GSoC](https://summerofcode.withgoogle.com/) and [RGSoC](https://railsgirlssummerofcode.org/).
```
Comment (Ana06) [inline README.md]: Do we want to add this in the main readme? :thinking:

File: CONTRIBUTING.md (hunk 5 above)
Comment (differentreality) [inline CONTRIBUTING.md]: I moved Contact and Code of conduct at the end of the file.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ChrisBr) [inline CONTRIBUTING.md]: It is not about first open an issue and then sending a PR. I want to avoid that people (especially GSoC students in preparation for GSoC) open random PR. Instead I want an issue, discussing in the issue if we want to implement it and finally a PR after we agreed on sth in the issue.

Comment (Ana06) [inline CONTRIBUTING.md]: Not pretty sure if you like what @differentreality wrote or you also want to change it :confused:

**Comment by Ana06** [comment]
I'll merge this because it is useful and we can discuss afterwards improving it :wink:
